lib/nx_zlib.h

```diff
@@ -332,6 +327,8 @@ static inline int use_nx_inflate(z_streamp strm)
 	/* #1 Threshold */
 	if(strm->avail_in <= DECOMPRESS_THRESHOLD) return 0;
 
+	if(nx_config.gzip_selector == GZIP_AUTO) return 1;
+
```

Comment:
>The way the code is written now, we'll use software zlib every time the input is too small

No, we already test for the other modes before testing the threshold.

>Note that if the user has selected GZIP_MIX, and the input is small, software zlib will be used 100% of the time, regardless of nx_ratio.

That is true, but this always been like that for `GZIP_MIX`, by design it wouldn't use the ratio for small inputs because software is a better choice in this case. I think this is a good approach and I will update the `GZIP_MIX` description to reflect that, but if you guys prefer to only rely on the ratio for that mode I'm open to discuss it.

Reply:
@rzinsly I think making the description more clear is enough. Thanks!